src/MultiStopGradient.js

```diff
@@ -21,9 +21,9 @@ const putInRange = (ranges, val) => {
 }
 
 const lineHeight = 24
-const Tick = ({value, colour, top}) =>
+const Tick = ({value, colour, top, position}) =>
   <div style={{position: `absolute`, height: `2px`, width: `20px`, background: colour, top: `${top + lineHeight}px`}}>
-    <div style={{position: `absolute`, marginLeft: `24px`, marginTop: `-${lineHeight / 2}px`}}>
+    <div style={{position: `absolute`, marginLeft: `${position === 'left' ? -24 : 24}px`, marginTop: `-${lineHeight / 2}px`}}>
```

Comment:
I remember you weren’t very happy with the 24 being there. At some point we could use [Sass](http://sass-lang.com/) (or Less, or whatever), and maybe we could move this sort of rules to the stylesheet itself.